File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Java/Parte1/Circulo.java
public class Circulo extends Figura {
    public Circulo(double radio) {
        super(radio);
    }

    @Override
    public double calcularArea() {
        return Math.PI * Math.pow(dimension1, 2);
    }

    @Override
    public double calcularPerimetro() {
        return 2 * Math.PI * dimension1;
    }
}

File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Python/Parte1/circulo.py
from figura import Figura
import math

class Circulo(Figura):
    def __init__(self, radio):
        super().__init__(radio)

    def calcular_area(self):
        return math.pi * self.dimension1 ** 2

    def calcular_perimetro(self):
        return 2 * math.pi * self.dimension1
